src/modules/content-tree/content.tree.module.js

```diff
@@ -15,6 +15,7 @@ export default class ContentTreeModule extends Component {
         this.handleCollapseAllItems = this.handleCollapseAllItems.bind(this);
         this.limitSubitemsInSubtree = this.limitSubitemsInSubtree.bind(this);
 
+        try {
```

Comment:
I decided to add try/catch in case anyone has corrupted saved subtree it will fallback to default. Are you fine with this, ping @dew326 ?

Reply:
Well, we should avoid saving the corrupted data in the first place. Why the corrupted data was saved?

Reply:
Corrupted data was saved because of previous error, which this PR fixes.